docs/kubernetes/kctlr-f5-resource.rst

```diff
@@ -0,0 +1,114 @@
+   ======================= ======================================================== ===========
+   Property                Description                                              Required
+   ======================= ======================================================== ===========
+   data                    A JSON object                                            Required
+   ----------------------- -------------------------------------------------------- -----------
+   - frontend              Defines BIG-IP objects.
+   ----------------------- -------------------------------------------------------- -----------
+   - backend               Identifies the Service you want to proxy.
+
+                           Defines a BIG-IP health monitor(s) for the Service.
+   ----------------------- -------------------------------------------------------- -----------
+   f5type                  A :code:`label` property watched by the |kctlr|.         Required
+   ----------------------- -------------------------------------------------------- -----------
+   schema                  Tells the |kctlr| how to interpret the encoded data.     Required
+   ======================= ======================================================== ===========
+
+.. _f5 resource frontend:
+
+Data.Frontend
+`````````````
+
+The :code:`data.frontend` property defines the objects you want to create on the BIG-IP system for a specific Service.
+In this section, you can :ref:`define a custom virtual server <kctlr-per-svc-vs>` using the :code:`virtualServer` properties or :ref:`deploy an iApp <kctlr-deploy-iapps>` using the :code:`iApp` properties.
+
+- `k8s-bigip-ctlr virtual server configuration parameters`_
+- `k8s-bigip-ctlr iApp configuration parameters`_
+
+The iApp configuration parameters include a set of customizable ``iappVariables`` parameters. These custom user-defined parameters must correspond to fields in the iApp template you want to launch. You can also define the `iApp pool member table`_ that the iApp creates on the BIG-IP system.
+
+.. tip:: With the iApp configuration parameters, you can also deploy objects across BIG-IP modules (for example, ASM and LTM).
+
+.. _f5 resource backend:
+
+Data.Backend
+````````````
+
+Use the :code:`data.backend` property to identify the `Kubernetes Service`_ that makes up the back end server pool.
+
+.. tip:: You can also define BIG-IP health monitors in this section.
+
+
+F5 type
+```````
+
+The usage of the :code:`f5type` property differs depending on whether you're using it in Kubernetes or OpenShift.
+
+- In an **F5 resource** ConfigMap, the :code:`f5type` value must be :code:`virtual-server`. This tells the |kctlr| that you want it to create a BIG-IP virtual server.
+
+- In an **OpenShift Route resource**, the :code:`f5type` property functions as a :code:`route-label` filter.
+  When you deploy the Controller with :code:`--route-label:true`, the |kctlr| watches the API for OpenShift Routes using the "f5type" label.
```

Comment:
The --route-label argument takes a string. This instructs the ctlr to filter on the provided string, for instance App1 in previous examples; therefore, the ctlr will filter Routes with the label 'f5type=App1'. Only the label value is required, the ctlr will always use the key 'f5type'.